src/v2/providers/alerts/index.ts

```diff
@@ -1 +1,5 @@
+import * as appDistribution from './appDistribution';
```

Comment:
nit* think this might be same as

```
export * as appDistribution from './appDistribution';
```

Also, why is this needed if we are explicitly declaring export path in package.json?

Reply:
removed since we declare export path in the package file